addons/mail/static/tests/new/messaging/messaging_tests.js

```diff
@@ -0,0 +1,93 @@
+/** @odoo-module **/
+
+let target;
+QUnit.module("messaging", {
+    async beforeEach() {
+        target = getFixture();
+    },
+});
+import { afterNextRender, start, startServer } from "@mail/../tests/helpers/test_utils";
+
+import { getFixture } from "@web/../tests/helpers/utils";
+
+QUnit.test(
+    "Posting a message to a partner out of discuss should open a chat window",
+    async function (assert) {
+        const pyEnv = await startServer();
+        const resPartnerId = pyEnv["res.partner"].create({ name: "Dumbledore" });
+        const mailChannelId = pyEnv["mail.channel"].create({
+            channel_member_ids: [
+                [0, 0, { partner_id: pyEnv.currentPartnerId }],
+                [0, 0, { partner_id: resPartnerId }],
+            ],
+            channel_type: "chat",
+        });
+        const [message] = pyEnv["mail.message"].searchRead([
+            [
+                "id",
+                "=",
+                pyEnv["mail.message"].create({
+                    author_id: resPartnerId,
+                    body: "You're a wizard, harry.",
+                    model: "mail.channel",
+                    res_id: mailChannelId,
+                }),
+            ],
+        ]);
```

Comment:
would probably gain a lot of lines out of prettier by saving the id of new message in a const

Reply:
10 lines, done :+1: